open_lm/params.py

```diff
@@ -576,7 +576,7 @@ def parse_args(args):
 
     if args.val_data is not None and args.val_batch_size is None:
         # if not set explicitly make sure that the val batch size is set to the micro batch size
-
-        args.val_batch_size = args.batch_size // args.accum_freq
+        # TODO: is this correct with global batch size?
```

Comment:
This part I'm not 100% sure about, I think it is assumed that we are running eval with only 1 GPU.

Reply:
I would do

```args.per_gpu_batch_size_val = (args.global_batch_size // world_size // args.accum_freq)```

and change references to `args.val_batch_size` to `args.per_gpu_batch_size_val` (I know, it's a mouthful). 

Other than that this LGTM!